src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -52,6 +52,7 @@
 #include "renderer/api/scene.h"
 #include "renderer/api/texture.h"
 #include "renderer/api/utility.h"
+#include "renderer/modeling/input/source.h"
```

Comment:
Why do you need this header file? In principle it's forbidden to include anything other than API headers.